src/documentation/chunk.rs

```diff
@@ -204,6 +199,16 @@ impl CheckableChunk {
     pub fn fragment_count(&self) -> usize {
         self.source_mapping.len()
     }
+
+    /// Obtain an accessor object containing mapping and string repr, removing the markdown anotations.
```

Comment:
this moved out from the original place but I guess is not important. I think happened to me but thought it was fixed it. only if you care...

Reply:
What do you mean? I don't quite get it :] I've seen that the rebase caused a duplication / removal so this might have been my bad but I did not see a negative impact here

Reply:
yeah, just obs but it is not relevant as did not affect anything rather than place.